### PR TITLE
T/clarify wording of user tracking in documentation

### PR DESCRIPTION
**Pull request by ziegenberg**
To prevent confusion of "logging" and "tracking":
* renamed admin documentation "Logging and Privacy" to "User Statistics and Privacy"
* replace "logging" with "tracking" to be more precise in the wording
* added note about Matomo module

### CODE REVIEW

File: docs/guides/admin/docs/configuration/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ General Configuration
  - [Database Configuration](database.md)
  - [Encoding Profile Configuration](encoding.md)
  - [Load Configuration](load.md)
- - [Logging and Privacy Configuration](logging.and.privacy.md)
+ - [Logging and Privacy Configuration](user-statistics.and.privacy.md)
```
Comment (lkiesow) [inline docs/guides/admin/docs/configuration/index.md]: Here, you probably want to use “User Statistics and Privacy” as well

Comment (ziegenberg) [reply docs/guides/admin/docs/configuration/index.md]: you suggest to rename the file to `user.statistics.and.privacy.md`?

Comment (ziegenberg) [reply docs/guides/admin/docs/configuration/index.md]: forget my last comment...

**Comment by ziegenberg** [comment]
rebased to current r/5.x

**Comment by lkiesow** [comment]
```
% mkdocs build
…
WARNING -  The page "configuration/index.md" contained a hyperlink to "configuration/logging.and.privacy.md" which is not listed in the "pages" configuration.
```

**Comment by ziegenberg** [comment]
fixed the issues

**Pull request merged**